terraform/paas/resources.tf

```diff
@@ -0,0 +1,104 @@
+resource "cloudfoundry_app" "find-teacher-training" {
+  for_each =  var.app
+
+  name         = each.value.name
+  space        = data.cloudfoundry_space.space.id
+  docker_image = each.value.docker_image
+ 
+  environment = {
+    ASSETS_PRECOMPILE                                = each.value.environment.ASSETS_PRECOMPILE
+    RAILS_ENV                                        = each.value.environment.RAILS_ENV
+    RAILS_SERVE_STATIC_FILES                         = each.value.environment.RAILS_SERVE_STATIC_FILES
+    SECRET_KEY_BASE                                  = var.SECRET_KEY_BASE
+    SENTRY_DSN                                       = var.SENTRY_DSN
+    SETTINGS__GOOGLE__GCP_API_KEY                    = var.SETTINGS__GOOGLE__GCP_API_KEY
+    SETTINGS__GOOGLE__MAPS_API_KEY                   = var.SETTINGS__GOOGLE__MAPS_API_KEY
+    SETTINGS__REDIRECT_RESULTS_TO_C_SHARP            = each.value.environment.SETTINGS__REDIRECT_RESULTS_TO_C_SHARP
+    WEBPACKER_DEV_SERVER_HOST                        = each.value.environment.WEBPACKER_DEV_SERVER_HOST
+    WEBSITE_SLOT_POLL_WORKER_FOR_CHANGE_NOTIFICATION = each.value.environment.WEBSITE_SLOT_POLL_WORKER_FOR_CHANGE_NOTIFICATION
+  }
+  timeout = 900
+  ports   = [3000]
```

Comment:
Can we double check? It's not [in the doc](https://github.com/cloudfoundry-community/terraform-provider-cf/wiki/resource_app)